README.md

```diff
@@ -42,54 +42,55 @@ Currently the launcher has quite limited in features – it allows you to launch
 
 As well as a CLI and GUI, we inject an API into all applications launched via the glu command (known as the `glu` function). All and any interactions with the operating system happens through this function which is bound to `window.glu`.
 
-The function takes a single argument – a command – in the form of a string, much like a terminal:
+The function takes a single argument – a command – in the form of a string, much like a terminal. Every time the `glu` function is called, a new connection between the browser window and a native child process is created. Calling the functions returns a promise of some buffered `stdout`:
 
 ```js
-glu('ls')(console.log);
+const version = await glu('node -v');
+// => v10.14.1
 ```
 
-Every time the `glu` function is called, a connection between the browser window and a native child process is created.
+This is especially convenient when working with commands that return a single value in a predictable amount of time.
 
-### glu(cmd)(callback)
+### glu(cmd, callback)
 
-The provided callback is repeatedly called with any output to `stdout` until the invoked process exits.
+For long running processes that might return multiple values before exiting, then provide a callback function. The callback function is called every time the child process prints to `stdout` or `stderr`. This happens until the child process exits.
 
 ```js
-glu('node -v')(stdout => console.log(stdout));
-
-// => v10.14.1
-```
-
-This works for synchronous processes that output just once (like `node -v`) but also for processes that take indeterminate amount of time to execute and potentially print to `stdout` multiple times. This is convenient for process like `ping`, `traceroute` or `npm install` for example.
-
-```js
-glu('ping google.com')(stdout => console.log(stdout));
+glu('ping google.com')(([stdout, stderr]) => {
```

Comment:
Should this be
`glu('ping google.com', ([stdout, stderr]) => { .. });`
?

Reply:
Yep, good spot.. all updated 👍